src/PrestaShopBundle/Translation/Provider/SearchProvider.php

```diff
@@ -32,6 +32,8 @@ class SearchProvider extends AbstractProvider implements UseDefaultCatalogueInte
 {
     private $domain;
 
+    private $modulesDirectory;
```

Comment:
this is not enough to use the search Provider but I have another idea ... Search Provider will reuse the list of every provider already registered until he got an instance of Catalogue.

If at the end of the list, still 0 catalogue => throw an exception.